PubSub/tests/System/ManageTopicsTest.php

```diff
@@ -77,4 +77,44 @@ public function testReloadTopic($client)
         $this->assertTrue($client->topic($shortName)->exists());
         $this->assertEquals($topic->name(), $topic->reload()['name']);
     }
+
+    /**
+     * @dataProvider clientProvider
+     */
+    public function testUpdateTopic($client)
+    {
+        $topics = $client->topics();
```

Comment:
Create a topic for the test rather than modifying an existing one. See the test immediately above for how to create a topic and enqueue it for cleanup at the end of the test run. (Same for the other system test you added)

Reply:
I don't mind using resources already setup by other tests. If we do that, though, it would be nice to use the `@depends` annotation and make sure the data pre-modification differs from the end result. In this case, it would probably be easier to just create a new resource.

Reply:
Updated the test to add new topics.